deeppavlov/core/commands/train.py

```diff
@@ -150,17 +152,28 @@ def train_model_from_config(config_path: str) -> None:
     except KeyError:
         log.warning('Train config is missing. Populating with default values')
 
-    metrics_functions = list(zip(train_config['metrics'],
-                                 get_metrics_by_names(train_config['metrics'])))
+    metrics_functions = list(zip(train_config['metrics'], get_metrics_by_names(train_config['metrics'])))
 
-    if callable(getattr(model, 'train_on_batch', None)):
-        _train_batches(model, iterator, train_config, metrics_functions)
-    elif callable(getattr(model, 'fit_batches', None)):
-        _fit_batches(model, iterator, train_config)
-    elif callable(getattr(model, 'fit', None)):
-        _fit(model, iterator, train_config)
-    elif not isinstance(model, Chainer):
-        log.warning('Nothing to train')
+    if not is_trained:
```

Comment:
Train if not is_trained?

Reply:
Yes, if the model is not trained we should train it, otherwise we use it for evaluation.

Reply:
We could want to retrain or train trained model on another dataset. How about `evaluate_mode`?